### PR TITLE
z-lua: init at 1.7.0

### PR DESCRIPTION
**Pull request by marsam**
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [X] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/misc/z-lua/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    patchShebangs z.lua
+  '';
+
+  installPhase = ''
+    install -Dm755 z.lua $out/bin/z
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/skywind3000/z.lua;
+    description = "jump around (lua implementation of z command for unix & windows)";
```
Comment (nlewo) [inline pkgs/tools/misc/z-lua/default.nix]:
```suggestion
    description = "Jump around (lua implementation of z command for unix & windows)";
```

**Review by nlewo — CHANGES_REQUESTED** [review]

File: pkgs/tools/misc/z-lua/default.nix (hunk 1 above)
Comment (marsam) [reply pkgs/tools/misc/z-lua/default.nix]: fixed

File: pkgs/tools/misc/z-lua/default.nix [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    rev = "581ae7a3b7b71267ecc15ec9a6a762397097d23a";
+    sha256 = "16nrprmzbx7kwc5ndbcvvxmajzf579yby3bdz4m86ff5vkc2g7av";
+  };
+
+  dontBuild = true;
+
+  buildInputs = [ lua ];
+
+  postPatch = ''
+    patchShebangs z.lua
```
Comment (Mic92) [inline pkgs/tools/misc/z-lua/default.nix]: This usually happens automatically.

Comment (marsam) [reply pkgs/tools/misc/z-lua/default.nix]: updated, thanks!

**Comment by Lassulus** [comment]
Thanks for the PR and sorry for the delay!
seems like there is a 1.6.0 now. Would be very cool if you could update it :)

**Comment by marsam** [comment]
no worries, updated :)

**Pull request merged**